pkgs/applications/misc/octoprint/default.nix

```diff
@@ -107,6 +107,24 @@ let
           }
         )
 
+        # Built-in dependency
+        (
+          self: super: {
+            file-read-backwards = self.buildPythonPackage rec {
```

Comment:
#136513 will add `file-read-backwards`

Reply:
Dropping file-read-backwards and octolapse from this PR